src/eval/methods.rs

```diff
@@ -56,9 +56,9 @@ pub fn call(
             "matches" => Value::Array(string.matches(args.expect("pattern")?)),
             "replace" => {
                 let pattern = args.expect("pattern")?;
-                let with = args.expect("replacement string")?;
+                let with = args.expect("function or string")?;
```

Comment:
```suggestion
                let with = args.expect("string or function")?;
```
the simpler thing should be first, in general